### PR TITLE
Feature/make command bar responsive

### PR DESCRIPTION
**Pull request by larisaVasile**

### CODE REVIEW

**Review by pavel-s-epam — APPROVED** [review]

File: src/ng-generate/components/shared/generators/components/command-bar/files/__name@dasherize__-command-bar.component.ts.template [hunk 1]
```diff
@@ -107,4 +124,54 @@ export class <%= classify(name) %>CommandBarComponent {
...
+        }
+    }
+
+    isInCollapsedMenu(id: string) {
+      return this.hiddenComponents.includes(id);
+    }
+
+    <% if (options.componentType === 'table' && options.hasSearchBar) { %>
+      openSettingsFromCollapsed() {      
+        const configMenuDialogRef =  this.dialog.open(MovementTableConfigMenuComponent, {
```
Comment (sha4be) [inline src/ng-generate/components/shared/generators/components/command-bar/files/__name@dasherize__-command-bar.component.ts.template:168]: the dialog component should be model-specific here 
```suggestion
        const configMenuDialogRef =  this.dialog.open(<%= classify(name) %>TableConfigMenuComponent, {
```

File: src/ng-generate/components/shared/generators/components/command-bar/files/__name@dasherize__-command-bar.component.ts.template [hunk 2]
```diff
@@ -107,4 +124,54 @@ export class <%= classify(name) %>CommandBarComponent {
...
+              data: { configs: this.configs, keyLocalStorage: this.KEY_LOCAL_STORAGE_MOVEMENT_TABLE_V100_CONFIG},
```
Comment (sha4be) [inline src/ng-generate/components/shared/generators/components/command-bar/files/__name@dasherize__-command-bar.component.ts.template:169]: The local storage key is dynamic, so I suggest the following change
```suggestion
              data: { configs: this.configs, keyLocalStorage: this.<%= options.localStorageKeyConfig %>},
```

File: src/ng-generate/components/shared/generators/components/command-bar/files/__name@dasherize__-command-bar.component.ts.template (hunk 1 above)
Comment (larisaVasile) [reply src/ng-generate/components/shared/generators/components/command-bar/files/__name@dasherize__-command-bar.component.ts.template]: done

File: src/ng-generate/components/shared/generators/components/command-bar/files/__name@dasherize__-command-bar.component.ts.template (hunk 2 above)
Comment (larisaVasile) [reply src/ng-generate/components/shared/generators/components/command-bar/files/__name@dasherize__-command-bar.component.ts.template]: done

**Review by sha4be — APPROVED** [review]

**Pull request merged**